packages/react-composites/src/composites/CallComposite/utils/Utils.ts

```diff
@@ -147,6 +165,12 @@ export const getCallCompositePage = (
           return 'joinCallFailedDueToNoNetwork';
         }
         return 'leftCall';
+      /* @conditional-compile-remove(rooms) */
+      case CallEndReasons.ROOM_NOT_FOUND:
+        return roomNotFoundPageTrampoline();
+      /* @conditional-compile-remove(rooms) */
+      case CallEndReasons.NOT_INVITED_TO_ROOM:
+        return notInvitedToRoomPageTrampoline();
```

Comment:
Do we need a trampoline here? The entire case statement will be removed (if not just remove return statement as well)